src/ds/ds-private.h

```diff
@@ -519,13 +524,16 @@ namespace librealsense
         enum gvd_fields
         {
             // Keep sorted
+            gvd_version_offset = 2,
             camera_fw_version_offset = 12,
             is_camera_locked_offset = 25,
             module_serial_offset = 48,
             module_asic_serial_offset = 64,
             fisheye_sensor_lb = 112,
             fisheye_sensor_hb = 113,
             imu_acc_chip_id = 124,
+            ip65_sealed_offset = 161,
+            filter_sensor_offset = 164,
```

Comment:
Maybe it is not the correct name "filter_sensor_offset"? Actually, a filter is not a sensor.

Reply:
I agree, please align to your previous name: `ir_filter_exist`

Reply:
Sorry, what do you mean by "previous name"?
In the struct, we have offsets of a memory.

Reply:
Like discussed

Reply:
Done